packages/admin/docs/04-dashboard.md

```diff
@@ -147,6 +147,51 @@ Alternatively, you may disable polling altogether:
 protected static ?string $pollingInterval = null;
 ```
 
+## Table widgets
+
+Filament comes with a "table" widget template, which you can use to display a table of data without needing to write a custom view.
+
+Start by creating a widget with the command:
+
+```bash
+php artisan make:filament-widget OrdersOverview
```

Comment:
Shouldn't this use the new `--table`flag instead of explaining how to replace the BaseWidget?